src/ucp/wireup/wireup_cm.c

```diff
@@ -737,13 +739,17 @@ static void ucp_cm_server_connect_cb(uct_ep_h ep, void *arg,
 {
     ucp_ep_h ucp_ep            = arg;
     uct_worker_cb_id_t prog_id = UCS_CALLBACKQ_ID_NULL;
-    ucp_lane_index_t wireup_idx;
+    ucp_lane_index_t cm_lane;
 
     if (status != UCS_OK) {
-        wireup_idx = ucp_ep_config(ucp_ep)->key.wireup_lane;
+        cm_lane = ucp_ep_get_cm_lane(ucp_ep);
+        if (status == UCS_ERR_REJECTED) {
+            ucp_ep->flags &= ~UCP_EP_FLAG_LOCAL_CONNECTED;
+            status = UCS_ERR_CONNECTION_RESET;
```

Comment:
why are we converting REJECTED to CONNECTION_RESET?

Reply:
this goes from rdmacm event, IMO REJECTED status is not obvious on server side :)
should we convert it on UCT/rdmacm level, WDYT?

Reply:
let's convert on UCT level

Reply:
Ok, will move to https://github.com/openucx/ucx/pull/4719